src/middlewares/isManager.js

```diff
@@ -0,0 +1,13 @@
+import setLanguage from '../utils/international';
+const isManager = (req, res, next) => {
+  const { role, preferedLang } = req.user;
+  if (role !== 'manager') {
+    return res.status(403).json({
+      status: 403,
+      error: setLanguage(preferedLang).__('notManager')
+    });
+  }
+  next();
+};
+
+export default isManager;
```

Comment:
I think there is a middleware that handles this already

Reply:
Sure @blaise82 . I did remove it